app/controllers/two_factor_authentication_controller.rb

```diff
@@ -7,9 +7,17 @@ class TwoFactorAuthenticationController < ApplicationController
   before_action :redirect_to_login_path, only: :verify, if: -> { @user.blank? }
   before_action :validate_user_id_from_params, except: :new
 
+  layout "inertia"
+
   # Get /two-factor
   def new
-    @hide_layouts = true
+    render inertia: "TwoFactorAuthentication/Index",
```

Comment:
```suggestion
    render inertia: "TwoFactorAuthentication/New",
```

Page component name must match the controller action (new -> New)